### DISCUSSION TITLE
Protein Quantification

### DISCUSSION THREAD

**Discussion by Josephineag** [vdemichev/DiaNN#266] [Q&A]
Hi, 
I am trying to understand how  DIA-NN passes form peptide (precursors) quantification to protein quantification.
![image](https://user-images.githubusercontent.com/44833227/145016498-0c67e080-eec3-4818-90f5-f6d218d260f6.png)

In the report, i highlighted the same protein group in 3 different samples. 
I noticed that the PG.Quantity mentioned for this protein comes from the peptide with the highest intensity (in red in the table). And for the same protein, the quantification comes from different peptides, the first two samples the peptide WNYGAL... was used and for the third sample the peptide GYFPEPVTV... was used. 

In the nature methods paper,  it is written that the 'top 3' method is used for protein quantification  (sum of the 3 most abundant precursors). However in my data is top 1. How can i change this parameter  ? And is there a parameter to fix the same peptides for quantification in all the samples. 

Thanks for your answer,
Josephine

**Comment by vdemichev** [comment]
Hi Josephine,

Yes, the 'regular' quantities reported by DIA-NN are Top 1 now, not Top 3, as Top 1 performs better. You can change this setting using --top [N], e.g. --top 3. But MaxLFQ is better than either Top 1 or Top 3, so I recommend just using the MaxLFQ quantities.

Best,
Vadim

**Reply by vdemichev** [reply]
Using same peptides for quantification shows very poor accuracy. So for Top N we choose the peptides (or a single peptide) separately for each run - this shows good accuracy in LFQBench-like tests. But MaxLFQ is better anyway.

**Reply by Josephineag** [reply]
Thanks for your answer, 
so if i understand correctly the MaxLFQ quantities are not based on the PG.Normalized or PG.quantity columns ?

**Reply by vdemichev** [reply]
Yes, MaxLFQ is directly calculated using precursor intensities with an implementation similar to https://linkinghub.elsevier.com/retrieve/pii/S1535947620333107 but with some changes

**Reply by Josephineag** [reply]
Thx for the publication, i will take a look,
Have a nice day and thx for the help
